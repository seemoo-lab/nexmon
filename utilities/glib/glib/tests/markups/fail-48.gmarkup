<fail foo="foo" bar
></foo>
