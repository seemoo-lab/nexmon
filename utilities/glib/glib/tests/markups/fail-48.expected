ERROR Error on line 2 char 2: Odd character '>', expected a '=' after attribute name 'bar' of element 'fail'
